4회 4주차/부족한 금액 계산하기.js

```diff
@@ -0,0 +1,10 @@
+function solution(price, money, count) {
+  let answer;
+  if (money < price * ((count * (count + 1)) / 2)) {
```

Comment:
```suggestion
  const totalCost = price * count * (count + 1) / 2;
```
if 문이 아닌 바로 계산하는 식으로 작성해도 좋을것 같아요 ㅎㅎ!